### DISCUSSION TITLE
Hello group....BillingTrack as a subfolder

### DISCUSSION THREAD

**Discussion by Kwaku-Ananse** [cytech/BillingTrack#50] [Q&A]
Hello, I was looking to be able to access BillingTrack from a subfolder and not a subdomain or full domain. Example www.mydomain.com/payme. I tried and the page didnt load well...no images loaded and page report not secure even though the domain www.mydomain .com is ssl secured. am using apache2 as backend and nginx as revers proxy....how can i work this around please?

**Comment by cytech** [comment]
there is a sample apache2 config here https://github.com/cytech/BillingTrack/wiki/Prerequisites.
aside from that you (or your IT professional) need to figure out how to implement this app in your configuration.
